libs/movex-core-util/src/lib/core-types.ts

```diff
@@ -202,7 +203,7 @@ export type GenericResourceOfType<TResourceType extends string> = Resource<
 
 export type GenericResourceType = GenericResource['type'];
 
-export type MovexClient<Info extends UnknownRecord = {}> = {
+export type MovexClient<Info extends UnknownRecord = Record<string, never>> = {
```

Comment:
Hey @Abhishek-Mallick, thank you for these changes. As you're already using `Record<string, never>` in 2 places here, and weit could most likely be used in other places as well, I suggest to create a new type `EmptyObject = Record<string, never>` and use that everywhere is needed!